lua/modules/MangaShiro.lua

```diff
@@ -294,7 +293,11 @@ function GetNameAndLink()
 			['41294a121062494489adfa601c442ef8'] = '/manga-list/?list', -- LegionAsia
 			['b53534f8443e420ea088594c53a3ff39'] = '/manhwa-list/', -- Manhwaland
 			['ff17b64aa945403dae45706753235872'] = '/latest-update/?list', -- KomikNesia
-			['5474e31b24ab4908a5258176d1f24f67'] = '/komik/list-mode/' -- ManhwaTaro
+			['5474e31b24ab4908a5258176d1f24f67'] = '/komik/list-mode/', -- ManhwaTaro
+			['f794803973af4e5daab21683d4de873a'] = '/series/list-mode/', -- LuminousScans
+			['86588503fd9e4277802c998cbccbc983'] = '/manga/list-mode/', -- AlphaScans
+			['275b85bdaafb47fdbc40f51d2bea99e8'] = '/manga/list-mode/' -- TheApolloTeam
```

Comment:
You can remove this as the main dirurl is already there at line 302.

Reply:
I did not see that, done.